frontend/src/components/editor/actions/useConfigActions.tsx

```diff
@@ -4,11 +4,9 @@ import { useAppConfig, useResolvedMarimoConfig } from "@/core/config/config";
 import type { AppConfig, UserConfig } from "@/core/config/config-schema";
 import { getAppWidths } from "@/core/config/widths";
 import { saveAppConfig, saveUserConfig } from "@/core/network/requests";
-import { useTheme } from "@/theme/useTheme";
 import type { ActionButton } from "./types";
 
 export function useConfigActions() {
-  const { theme } = useTheme();
```

Comment:
I think it would be nice if we actually used the `theme` below to set the optional `hidden` property on each of the options. That way, we could show only the other options when someone types in "theme". For example:

```ts
    {
      label: "Config > Set theme: light",
      hidden: theme === "light",
      handle: () => {
        handleUserConfig({
          ...config,
          display: {
            ...config.display,
            theme: "light",
          },
        });
      },
    },
```